### PR TITLE
Added the 5 IR @ Home Autonomous Challenges

### LINKED ISSUE #199: Current IR @ Home Challenges are not Available as Field Images
**Is your feature request related to a problem? Please describe.**  
Yes not having the image makes it much harder to make a useful path.

**Describe the solution you'd like**  
The field images will show up as games see my PR

### PR DESCRIPTION
**Pull request by jasondaming**
Closes #199

### CODE REVIEW

**Comment by AustinShalit** [comment]
This could be confusing if the paths change in the future. What do you think about including a general 2021 at home image with all the points on it (like there is at the start of the game manual)?

**Comment by jasondaming** [comment]
If the paths change in the future?  When do you think the paths will change?  Why would they change?  Seems like a low likelyhood of change and if they did why wouldn't we just update this?  

I actually started with a single general image that looked like the initial field setup at the top of page 16 and found it hard to use.

**Comment by AustinShalit** [comment]
Good point. How about we just add the year to the front of each image (`2021-barrelracingpath.json`)?

**Comment by jasondaming** [comment]
Okay I like that as it does improve consistency with the other things that are already there.

File: src/main/java/edu/wpi/first/pathweaver/Game.java [hunk 1]
```diff
@@ -16,6 +16,11 @@
   public static final Game INFINTE_RECHARGE_2020 = loadGameFromResource("2020-infiniterecharge.json");
   public static final Game DEEP_SPACE_2019 = loadGameFromResource("2019-deepspace.json");
   public static final Game POWER_UP_2018 = loadGameFromResource("2018-powerup.json");
+  public static final Game BARREL_RACING_PATH = loadGameFromResource("barrelracingpath.json");
```
Comment (sciencewhiz) [inline src/main/java/edu/wpi/first/pathweaver/Game.java:19]: need to update these names

Comment (jasondaming) [reply src/main/java/edu/wpi/first/pathweaver/Game.java]: thanks!

File: src/main/java/edu/wpi/first/pathweaver/Game.java [hunk 2]
```diff
@@ -16,11 +16,11 @@
...
-  public static final Game BARREL_RACING_PATH = loadGameFromResource("barrelracingpath.json");
-  public static final Game BOUNCE_PATH = loadGameFromResource("bouncepath.json");
-  public static final Game GALACTIC_SEARCH_A = loadGameFromResource("galacticsearcha.json");
-  public static final Game GALACTIC_SEARCH_B = loadGameFromResource("galacticsearchb.json");
-  public static final Game SLALOM_PATH = loadGameFromResource("slalompath.json");
+  public static final Game BARREL_RACING_PATH = loadGameFromResource("2021-barrelracingpath.json");
+  public static final Game BOUNCE_PATH = loadGameFromResource("2021-bouncepath.json");
+  public static final Game GALACTIC_SEARCH_A = loadGameFromResource("2021-galacticsearcha.json");
+  public static final Game GALACTIC_SEARCH_B = loadGameFromResource("2021-galacticsearchb.json");
+  public static final Game SLALOM_PATH = loadGameFromResource("2021-slalompath.json");
```
Comment (AustinShalit) [inline src/main/java/edu/wpi/first/pathweaver/Game.java:23]: The variable names too please

Comment (jasondaming) [reply src/main/java/edu/wpi/first/pathweaver/Game.java]: no probelm

**Comment by sciencewhiz** [comment]
The coordinates in the pictures do not match Pathweaver coordinates (but do match the coordinates for WPILib Trajectories after export). Not sure if that will be confusing.

**Comment by jasondaming** [comment]
I am not sure what you mean.  Are you talking about the markings in ft on the outside of the pictures?  Or are you talking about how PathWeaver goes 0,-27 in the Y direction (but is okay after export) if the latter then that is a PathWeaver thing and not related to this PR

**Comment by sciencewhiz** [comment]
Y=0 in pathweaver is upper left and lower left is y=-15. In the picture y=0 is lower left and upper left is y=15.

**Comment by jasondaming** [comment]
Interesting is that something to do with the picture or how / why does that happen?

**Comment by sciencewhiz** [comment]
I'm talking about the coordinates built into the pictures. 

image

**Comment by jasondaming** [comment]
I understand now better thank you.  It didn't really confuse me as I guess I didn't use the waypoint X and Y values except to tune them relative to each other (and other X and Y) and never place a point with just X and Y.  I might be able to Paint.net some different labels if you think it is essential.  If we do that however I would probably want to convert to meters as I think that is what most people are using in PathWeaver?  That would break the nice pretty divisions.  Not sure what is best.

**Comment by PeterJohnson** [comment]
My recommendation would be to just remove the coordinates in the image entirely (e.g. just leave the "field" portion).

**Comment by jasondaming** [comment]
Anyone else have thoughts / input?  If we get consensus I can do that

**Comment by PeterJohnson** [comment]
Yes, we have consensus on the team for that approach.

**Comment by sciencewhiz** [comment]
I think removing the coordinates is better then having bad coordinates and is consistent with the other field images.

**Comment by sciencewhiz** [comment]
By cropping so close, a waypoint in the corner of the field doesn't get fully shown. This might not be a problem except that the default path has one. You can still select it, however, if someone tries.
java_taPyf0k4tY

**Comment by jasondaming** [comment]
To me it seems like a PathWeaver issue but I guess I could crop it with some amount of border and resize the field if that is what the team wants.  How much border if that is what we want?

**Comment by PeterJohnson** [comment]
It's actually an issue in Field2d as well.  Maybe 10-20 pixels of border?

**Comment by jasondaming** [comment]
I added a 20 pixel border hope that looks good to everyone

**Review by sciencewhiz — APPROVED** [review]

**Review by PeterJohnson — APPROVED** [review]

**Pull request merged**